### PR TITLE
add druid adapter

### PR DESCRIPTION
**Pull request by byWeaponLin**
add druid adapter

### CODE REVIEW

File: flexy-druid/src/test/resources/spring/applicationContext-test.xml [hunk 1]
```diff
@@ -0,0 +1,48 @@
...
+		http://www.springframework.org/schema/context http://www.springframework.org/schema/context/spring-context-3.0.xsd">
+
+  <context:component-scan base-package="com.vladmihalcea.flexypool.config"/>
+  <!-- Xml configuration -->
+  <!--<import resource="classpath:spring/applicationContext-xml-config.xml"/>-->
+
+  <import resource="classpath:spring/applicationContext-local-resource.xml"/>
+
+  <bean id="poolingDataSource" class="com.alibaba.druid.pool.DruidDataSource" destroy-method="close">
+    <!-- 基本属性 url、user、password -->
```
Comment (vladmihalcea) [inline flexy-druid/src/test/resources/spring/applicationContext-test.xml]: Can you try to translate the Chinese description to English? Thanks.

**Comment by vladmihalcea** [comment]
Can you take a look at Travis? I see some [failures in your tests](https://travis-ci.org/vladmihalcea/flexy-pool/jobs/305266490#L7590).

File: flexy-druid/src/test/resources/spring/applicationContext-test.xml (hunk 1 above)
Comment (byWeaponLin) [reply flexy-druid/src/test/resources/spring/applicationContext-test.xml]: Sorry, I forgot it.

**Comment by vladmihalcea** [comment]
Applied upstream, thanks

**Pull request closed without merging**